src/models/Card/CardOptions.ts

```diff
@@ -0,0 +1,4 @@
+export interface CardOptions {
+    status?: string;
+    searchTerm?: string;
+}
```

Comment:
Doesn't appear to be used outside of the card class so move this to the card file and make it "internal"?